testing/wicket-threadtest/src/site/site.xml

```diff
@@ -16,9 +16,4 @@
    limitations under the License.
 -->
 <project name="Wicket Thread Test">
-	<skin>
-		<groupId>org.apache.wicket</groupId>
-		<artifactId>wicket-site-skin</artifactId>
-		<version>1.0-SNAPSHOT</version>
-	</skin>
```

Comment:
I have no idea what that is!

Reply:
I guess long time ago the site was generated by maven .... :)))